content/_help/verify-your-identity/verify-your-identity-in-person._en.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: help
+title: Verify your identity in person
+category: verify-your-identity
```

Comment:
I think we can add a `permalink` to resolve the issue with the path including the `._en`:

```suggestion
category: verify-your-identity
permalink: /help/verify-your-identity/verify-your-identity-in-person/
```

Reply:
Fixed in 265031a.